.github/workflows/ci.yaml

```diff
@@ -17,8 +17,7 @@ jobs:
             os: ubuntu-22.04
             compiler: gcc
             version: 12
-            # We exclude @freetype2//:bbox_test as that can take >1 minute to run under valgrind in CI.
-            bazel: --run_under="valgrind --leak-check=full --errors-for-leak-kinds=all --error-exitcode=1 --track-origins=yes --show-leak-kinds=all" -- -@freetype2//:bbox_test
+            bazel: --test_timeout=120 --run_under="valgrind --leak-check=full --errors-for-leak-kinds=all --error-exitcode=1 --track-origins=yes --show-leak-kinds=all"
```

Comment:
it's yaml; might as well make this a variable... given it'll probably need a bigger increase soon

Reply:
This is close to where it's used, and nothing else is anywhere remotely close to hitting the 60s timeout we had before. Leaving it as-is.